app/views/content/teach-disabled-pupils-and-pupils-with-special-educational-needs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Teach disabled pupils and pupils with special educational needs"
+description: |-
+  As a qualified teacher you’ll have the opportunity to work with pupils with complex needs, no matter what setting you work in.
+date: "2021-05-07"
+image: "/assets/images/my-story-hero-dt.jpg"
+article_classes: ['longform']
+keywords:
+  - SEND
+  - disabled
+  - disability  
+  - disabilities
+  - special educational needs
+  - teaching
+  - teacher
+  - special school
+---
+Most disabled pupils and pupils with special educational needs learn in mainstream schools.
+
+No matter what settings you train and work in, whether it's mainstream schools or special schools or both, you’ll work with pupils with complex needs.
```

Comment:
maybe I just love commas, but would

> whether it's mainstream schools, special schools, or both, 

work better?

Reply:
```suggestion
No matter what settings you train and work in – whether it's mainstream schools, special schools, or both – you’ll work with pupils with complex needs.
```

Reply:
in which case I'd use en rules to add that bit in as a separate clause - otherwise comma overload!